webpack.config.js

```diff
@@ -62,7 +63,8 @@ module.exports = {
       "selectors": absolute("./content-src/selectors"),
       "lib": absolute("./content-src/lib"),
       "strings": absolute("./strings"),
-      "test": absolute("./content-test")
+      "test": absolute("./content-test"),
+      "l20n": absolute("./node_modules/l20n/dist/compat/web/l20n.js")
```

Comment:
I know, super annoying. :( we could also try specifically excluding it from the optimizer 

Can't wait til that harmony branch lands in uglify...